src/GovUk.Education.ExploreEducationStatistics.Public.Data.Processor.Tests/Functions/CompleteNextDataSetVersionImportFunctionTests.cs

```diff
@@ -0,0 +1,365 @@
+using GovUk.Education.ExploreEducationStatistics.Common.Extensions;
+using GovUk.Education.ExploreEducationStatistics.Common.Tests.Extensions;
+using GovUk.Education.ExploreEducationStatistics.Content.Model;
+using GovUk.Education.ExploreEducationStatistics.Content.Model.Database;
+using GovUk.Education.ExploreEducationStatistics.Content.Model.Tests.Fixtures;
+using GovUk.Education.ExploreEducationStatistics.Public.Data.Model;
+using GovUk.Education.ExploreEducationStatistics.Public.Data.Model.Database;
+using GovUk.Education.ExploreEducationStatistics.Public.Data.Model.Tests.Fixtures;
+using GovUk.Education.ExploreEducationStatistics.Public.Data.Processor.Functions;
+using GovUk.Education.ExploreEducationStatistics.Public.Data.Processor.Model;
+using GovUk.Education.ExploreEducationStatistics.Public.Data.Processor.Requests;
+using GovUk.Education.ExploreEducationStatistics.Public.Data.Processor.Requests.Validators;
+using GovUk.Education.ExploreEducationStatistics.Public.Data.Processor.ViewModels;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.DurableTask;
+using Microsoft.DurableTask.Client;
+using Microsoft.EntityFrameworkCore;
+using Moq;
+using static GovUk.Education.ExploreEducationStatistics.Common.Tests.Utils.MockUtils;
+using FileType = GovUk.Education.ExploreEducationStatistics.Common.Model.FileType;
+
+namespace GovUk.Education.ExploreEducationStatistics.Public.Data.Processor.Tests.Functions;
+
+public abstract class CompleteNextDataSetVersionImportFunctionTests(
+    ProcessorFunctionsIntegrationTestFixture fixture)
+    : ProcessorFunctionsIntegrationTest(fixture)
+{
+    public class CompleteNextDataSetVersionImportTests(
+        ProcessorFunctionsIntegrationTestFixture fixture)
+        : CompleteNextDataSetVersionImportFunctionTests(fixture)
+    {
+        [Fact]
+        public async Task Success()
```

Comment:
Maybe we could add some checks for:
- the other viewmodel properties of `responseViewModel`, to make sure they've all been set properly
- grab the import from the DB, and make sure the status has been updated to `Completing`
- grab the data set version from the DB, and make sure the status has been updated to `Draft`

Reply:
So this test doesn't actually run any of the functions in the orchestration - it just verifies that the orchestration runs through the correct list of Functions, and any updates prior to running the orchestration have been made (e.g. creating an empty DataSet before running the "initial data set version" orchestration), so there are no other db updates that would be looked for in this particular test.

I have however added in checks for the other viewmodel fields as suggested, ta.